Source/Core/DolphinQt2/Settings.h

```diff
@@ -17,12 +18,12 @@ enum class Language;
 class InputConfig;
 
 // UI settings to be stored in the config directory.
-class Settings final : public QSettings
+class Settings final : public QSettings, public NonCopyable
```

Comment:
`public` should be omitted from the NonCopyable here so this is private inheritance. Since this is a convenience trait class, it shouldn't have an effect on the underlying type hierarchy. (i.e. deleting the copy constructors/assignment operators manually doesn't produce a new type in a hierarchy that can be used, so neither should this).